analysis/applySelections.cpp

```diff
@@ -173,19 +217,40 @@ int main(int argc, char* argv[])
     outFilePaths.push_back(argv[7]); // LbL signal extraction
     outFilePaths.push_back(argv[8]); // QED signal extraction
   }
+
   if(argc == 5){
     configPath = argv[1];
     inFilePath = argv[2];
     outFilePaths.push_back(argv[3]);
     outFilePaths.push_back(argv[4]);
   }
+  
   if(argc == 4){
     configPath = argv[1];
     inFilePath = argv[2];
     outFilePaths.push_back(argv[3]);
   }
+
+    if(argc == 3){
+    std::string arg_1 = argv[1];
+    std::string arg_2 = argv[2];
```

Comment:
Here and in general: no need for `std::`, as we have a global `using namespace std` in Helpers.hpp

Also, please give some meaningful names to variables, e.g. arg_2 -> setupFilePath